src/pages/pagesData.tsx

```diff
@@ -0,0 +1,35 @@
+import { HomePage } from './home-page/HomePage';
+import { CreateCard } from './create-card/CreateCard';
+import { UpdateCard } from './update-card/UpdateCard';
+import { About } from './about/About';
+
+export interface RouterType {
+  path: string;
+  element: JSX.Element;
+  title: string;
+}
```

Comment:
You don't need a custom interface for `RouteType` which you use as RouterProps, you already had it as `import { RouteProps } from 'react-router';`
So you can see the title is an unnecessary prop because the path is already a unique prop that you can use for key value for your `Route cmp`.